circuits/src/main.nr

```diff
@@ -1,13 +1,15 @@
 mod padding;
+mod permutations;
 
 // TODO: Change to final value (<= BLOCK_SIZE - 2).
-global INPUT_SIZE: Field = 10;
+global INPUT_SIZE: Field = 1088;
```

Comment:
```suggestion
global INPUT_SIZE: Field = 1086;
```

This value needs to be at least 2 less than BLOCK_SIZE otherwise we don't write the padding bits in correctly.

Reply:
https://github.com/TomAFrench/keccak256-noir/issues/5